admin/ui/route.go

```diff
@@ -91,7 +91,7 @@ $(function(){
 			tbl += '<td>' + (i+1) + '</td>';
 			tbl += '<td>' + r.service + '</td>';
 			tbl += '<td>' + r.src + '</td>';
-			tbl += '<td>' + r.dst + '</td>';
+			tbl += '<td><a href="' + r.dst + '">' + r.dst + '</a></td>';
```

Comment:
Does the attribute have to be escaped? What happens if r.dst contains a quote, for instance?.

Reply:
I suppose this code is already vulnerable to XSS. I'm gonna fix this in another PR.

Reply:
I've only seen `dst` be an ip:port combo, not sure how clicking on a link has anything to do with XSS?

Reply:
`dst` is an arbitrary string from the configuration and may contain `<script>evilCode();</script>`. But again, I've addressed this in #588, so don't worry about it here.